Xamarin.Forms.Xaml/MarkupExtensionParser.cs

```diff
@@ -18,6 +18,10 @@ public object Parse(string match, ref string remaining, IServiceProvider service
 				markupExtension = new TemplateBindingExtension();
 			else if (match == "StaticResource")
 				markupExtension = new StaticResourceExtension();
+			else if (match == "OnPlatform")
+				markupExtension = new OnPlatformExtension();
+			else if (match == "OnIdiom")
+				markupExtension = new OnIdiomExtension();
```

Comment:
this isn't required, but it allows bypassing constructing the markup through reflection